doc/source/new-project-guide/level-4-tutorial.rst

```diff
@@ -221,7 +221,7 @@ Trigger pre-commit hooks automatically with Git commit
 
     .. code-block:: bash
 
-        git add LICENSE
+        git add .
```

Comment:
@cadenmyers13 under the current tutorial, it's `LICENSE` which has been tested by yucong and zhiming. Let's first address the comments brought by @sbillinge